db/migrate/20170519181110_add_description_to_collections.rb

```diff
@@ -0,0 +1,6 @@
+class AddDescriptionToCollections < ActiveRecord::Migration
+  def change
+    add_column :collections, :description, :text, default: ""
```

Comment:
Easiest way is to add the column without a default, backfill it using a rake task / migration and then enforce a default value in a different migration. My understanding is that adding a column with a default will force the table to be copied to a new location on disk (with the new col defaults) and lock while it does so.